pkg/featureflag/featureflag.go

```diff
@@ -88,7 +88,7 @@ var (
 	// ClusterAddons activates experimental cluster-addons support
 	ClusterAddons = new("ClusterAddons", Bool(false))
 	// UseServiceAccountIAM controls whether we use pod-level IAM permissions for our system pods and kOps addons.
-	UseServiceAccountIAM = new("UseServiceAccountIAM", Bool(false))
+	UseServiceAccountIAM = new("UseServiceAccountIAM", Bool(true))
```

Comment:
grep shows that this isnt used anymore so I think we can remove it as well as from `./tests/e2e/scenarios/lib/common.sh`

Reply:
Unknown feature flags are logged and otherwise ignored, so this should be removed.

Reply:
Done.